google/cloud/sql/connector/InstanceConnectionManager.py

```diff
@@ -54,14 +64,29 @@ class InstanceConnectionManager:
     # functionality on Windows. It is recommended to use ProactorEventLoop
     # while developing on Windows.
     _loop: asyncio.AbstractEventLoop = None
+    _mutex: threading.Lock = None
+
+    _client_session: aiohttp.ClientSession = None
+    _credentials: Credentials = None
+
+    
     _instance_connection_string: str = None
+    _instance: str = None
     _project: str = None
     _region: str = None
-    _instance: str = None
-    _credentials: Credentials = None
+
+    _metadata: Dict[str, Union[Dict, str]] = None
+
     _priv_key: str = None
     _pub_key: str = None
 
+    _current: concurrent.futures.Future = None
+    _next: concurrent.futures.Future = None
+
+    _delay: int = 15
+
+    _logger: logging.Logger = None
```

Comment:
This can be a module level variable. 

Each manager_instance is using the same name, so it's actually returning the same object. It is deceptive to assign it when the manager_instance object is created. 

Just put this (after imports and before class):
```python
logger = logging.getLogger(__name__)
```

then log like this:

```python
logger.debug("So python much idiomatic")
```

Even inside the class - the variable is module wide so all of the manager_instances will use the same one.